subsys/fs/littlefs_fs.c

```diff
@@ -882,6 +890,12 @@ static int littlefs_init_cfg(struct fs_littlefs *fs, int flags)
 		lcp->sync = lfs_api_sync;
 	}
 
+#ifdef CONFIG_FS_LITTLEFS_DISK_VERSION
+	lcp->disk_version = disk_version;
+	LOG_INF("partition disk version: %u.%u", LFS_DISK_VERSION_MAJOR,
```

Comment:
I think it's better to change `LFS_DISK_VERSION_MAJOR` by `FS_LITTLEFS_DISK_VERSION_MAJOR_GET(disk_version)`.

Reply:
I also think so, you may also report here what is actual driver supported version.

Reply:
@de-nordic, the `littlefs_init_fs` function reports the actual supported disk version. Will this be enough?

Reply:
Yes.